ios/RNFIRMessaging.m

```diff
@@ -170,6 +176,16 @@ + (void)willPresentNotification:(UNNotification *)notification withCompletionHan
     [[NSNotificationCenter defaultCenter] postNotificationName:FCMNotificationReceived object:self userInfo:@{@"data": data, @"completionHandler": completionHandler}];
 }
 
++ (void)didRegisterUserNotificationSettings:(__unused UIUserNotificationSettings *)notificationSettings
```

Comment:
Not easily.  This connects the class method to the instance method via broadcast.  One could convert to a singleton perhaps?  And this fits with how other things are done here. Am I missing something?